pkg/olog/default_handler.go

```diff
@@ -25,7 +25,7 @@ var (
 	defaultHandler atomic.Int32
 
 	// defaultOut is the default output for the default handler. This is
-	// set to `os.Stderr` by default.
+	// set to `os.Stderr` by default. Can be update by SetOutput()
```

Comment:
Nit: docs formatting

```suggestion
	// set to `os.Stderr` by default. Can be updated by [SetOutput].
```